src/main/cpp/Robot.cpp

```diff
@@ -0,0 +1,188 @@
+// Copyright (c) 2017-2020 FRC Team 3512. All Rights Reserved.
+
+#include "Robot.hpp"
+
+#include <cmath>
+
+Robot::Robot() {
+    m_autonSelector.AddAutoMethod(
+        "Noop Auton", [] {}, [] {});
+
+    m_autonSelector.AddAutoMethod(
+        "DriveForward Autonomous",
+        std::bind(&Robot::DriveForwardAutonInit, this),
+        std::bind(&Robot::DriveForwardAuton, this));
+
+    m_autonSelector.AddAutoMethod("Right/Left Autonomous",
+                                  std::bind(&Robot::RightLeftAutonInit, this),
+                                  std::bind(&Robot::RightLeftAuton, this));
+
+    m_autonSelector.AddAutoMethod(
+        "MotionProfile", std::bind(&Robot::AutonMotionProfileInit, this),
+        std::bind(&Robot::AutonMotionProfile, this));
+
+    m_autonSelector.AddAutoMethod("Side Auton",
+                                  std::bind(&Robot::SideAutonInit, this),
+                                  std::bind(&Robot::SideAuton, this));
+
+    displayTimer.Start();
+}
+
+void Robot::DisabledInit() {
+    claw.SetAngle(0);
+    claw.ManualSetAngle(0);
+    claw.SetWheelManual(0);
+}
+
+void Robot::DisabledPeriodic() {
+    while (IsDisabled()) {
+        claw.Update();
+
+        frc2::Wait(0.1_s);
+        // std::cout << "Drive Stick 1: "<< driveStick1.GetY() << std::endl;
+        // std::cout << "Drive Stick 2: "<< driveStick2.GetZ() << std::endl;
+    }
+}
+
+void Robot::RobotPeriodic() {}
+
+void Robot::AutonomousInit() {
+    autonTimer.Reset();
+    autonTimer.Start();
+}
+
+void Robot::AutonomousPeriodic() { autonTimer.Stop(); }
```

Comment:
You probably didn't intend to call autonTimer.Stop() every 20ms. You also aren't actually running any autonomous modes. Look at how Robot-2015 does it.